app/src/main/java/com/wearerommies/roomie/presentation/ui/mypage/name/NameState.kt

```diff
@@ -0,0 +1,11 @@
+package com.wearerommies.roomie.presentation.ui.mypage.name
+
+import com.wearerommies.roomie.domain.entity.TourEntity
+
+data class NameState(
+    //todo: entity 변경
+    val uiState: TourEntity = TourEntity(),
+    val isValidated: Boolean = true
+) {
+    val isEnabled = uiState.name.isNotEmpty()
+}
```

Comment:
```suggestion
data class NameState(
    //todo: entity 변경
    val uiState: TourEntity = TourEntity(),
    val isValidated: Boolean = true
) {
    val isEnabled = isValidated && uiState.name.isNotEmpty()
}
```


만약 validated 는 텍스트 필드 값에만 의존하고, enabled는 valid여부에 따라서 + empty 여부에 따라서 결정될 필요가 있어보여요! (개인적 의견일 수 있음)
저라면 이런 방법으로고 구현해볼 것 같습니다 :)